docs/use-service.md

```diff
@@ -34,6 +34,23 @@ Here's a sample tile in Normal format:
 http://tile.mapzen.com/mapzen/terrain/v1/normal/11/330/790.png?api_key=your-mapzen-api-key
 ```
```

Comment:
We should add a note in the **Requesting tiles** section that all releases in the 1.x series are available in the `v1/` file path, minor update versioning isn't available via the Mapzen API... something something.

Reply:
I wonder if we should not mention 1.1 at all and instead say that we updated it rather than confuse with a minor version number?

Reply:
"data pack 2017". You're probably right.